File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/dart/practicas_3/preferir.dart
import 'dart:io';

void main () {

  int thing1 = 0;
  int thing2 = 0;

  print('¿Cuantas personas van a participar en la encuenta?');
  int cuantos = int.parse(stdin.readLineSync()!);

  for (int vueltas = 1; vueltas <= cuantos; vueltas++) {

    print('----------------------------------------------------------');

    print('¿Que prefieres mas tener superfuerza o supervelocidad?');

    print('----------------------------------------------------------');

    String choose = stdin.readLineSync()!;

    if (choose == 'superfuerza'){
      thing1 = thing1 + 1;
    } else if (choose == 'supervelocidad'){
      thing2 = thing2 + 1;
    }

    print('Respuesta registrada correctamente');

  }

  print('----------------------------------------------------------');

  print('RESULTADO DE LA ENCUESTA:');

  print('----------------------------------------------------------');

  print('Personas que escogieron superfuerza:');
  print(thing1);

  print('Personas que escogieron supervelocidad:');
  print(thing2);

  print('----------------------------------------------------------');

}

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/ruby/practicas_3/preferir.rb
thing1 = 0
thing2 = 0

puts "¿Cuantas personas van a participar en la encuenta?"
cuantos = gets.chomp.to_i

for vueltas in 1..cuantos

    puts "----------------------------------------------------------"

    puts "¿Que prefieres mas tener superfuerza o supervelocidad?"

    puts "----------------------------------------------------------"

    choose = gets.chomp

    if choose == "superfuerza"
      thing1 = thing1 + 1
    elsif choose == "supervelocidad"
      thing2 = thing2 + 1
    end

    puts "Respuesta registrada correctamente"

end

puts "----------------------------------------------------------"

puts "RESULTADO DE LA ENCUESTA:"

puts "----------------------------------------------------------"

puts "Personas que escogieron superfuerza:"
puts thing1

puts "Personas que escogieron supervelocidad:"
puts thing2

puts "----------------------------------------------------------"